perftestnotebook/transformer.py

```
@@ -131,3 +134,41 @@ def merge(self, sde):
 
         self.entry_number = 0
         return merged
+
+
+def get_transformers(filepath="perftestnotebook/customtransforms"):
+    """
+    This function returns a dict of transformers under the given path.
+
+    If more than one transformers have the same class name, only one of them will be selected.
```

We should raise an error if there are multiple classes with the same name. Otherwise, the user might think everything is fine when it isn't.